apps/web/src/views/Mev/hooks/index.ts

```diff
@@ -76,7 +69,9 @@ export const useAddMevRpc = (onSuccess?: () => void, onBeforeStart?: () => void,
           console.info('RPC network added successfully!')
           onSuccess?.()
         } catch (error) {
-          console.error('Error adding RPC network:', error)
+          if ((error as any).code === -32601) {
```

Comment:
we need to pre-detect before actually asking users to add the network?

Reply:
also let's use wallet client instead of `window.ethereum`

Reply:
got it, let me make the adjustment

Reply:
> we need to pre-detect before actually asking users to add the network?

yeah, should have already done it in useIsMEVEnabled hook, let me move the error message to that hook